### PR TITLE
Add Ixora keypad

### PR DESCRIPTION
**Pull request by coarse**
## Description

Adds Ixora config to QMK  
Note: `keyboards/peiorisboards` will house all my configs for my previous boards, migration of other configs will be in another PR to keep things simple

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/peiorisboards/ixora/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ * the keyboard. They are best used in combination with the HID Listen program,
+ * found here: https://www.pjrc.com/teensy/hid_listen.html
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
+/* key combination for magic key command */
+#define IS_COMMAND() ( \
```
[the remark this answers is not in the document]
Comment (coarse) [reply keyboards/peiorisboards/ixora/config.h]: Commented out on latest commit

File: keyboards/peiorisboards/ixora/ixora.h [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#ifndef IXORA_H
+#define IXORA_H
+
+#define XXX KC_NO
+
+#include "quantum.h"
+
+// This a shortcut to help you visually see your layout.
+
+#define LAYOUT_FULL( \
```
[the remark this answers is not in the document]
Comment (coarse) [reply keyboards/peiorisboards/ixora/ixora.h]: I've changed already changed this on all files affected. Please check latest commit to see changes 😄

File: keyboards/peiorisboards/ixora/config.h [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ * the keyboard. They are best used in combination with the HID Listen program,
+ * found here: https://www.pjrc.com/teensy/hid_listen.html
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
+/* key combination for magic key command */
+// #define IS_COMMAND() ( \
```
Comment (drashna) [inline keyboards/peiorisboards/ixora/config.h]: This is actually generating compiler errors. 

Please remove the entire `IS_COMMAND` block.

Comment (coarse) [reply keyboards/peiorisboards/ixora/config.h]: Weird, it's working on my msys environment. Anyway, I've already deleted this part of the code as well as any other commented option that wasn't being used in the latest commit.

Comment (drashna) [reply keyboards/peiorisboards/ixora/config.h]: Yeah, it was having issues with the `\` in Travis.  Multi-line comment error, or some such.   

Super weird.   But thanks!

**Pull request merged**